exercises/isogram/src/Isogram.hs

```diff
@@ -0,0 +1,6 @@
+module Isogram (isIsogram) where
+
+import Data.Char
```

Comment:
since

1. Not every solution wants or needs this
2. I wouldn't want to encourage the habit of importing everything rather than the specific list of functions desired

I would remove this line